packages/squiggle-lang/src/js/cli/utils.ts

```diff
@@ -0,0 +1,71 @@
+import { SqProject } from "../SqProject";
+import { SqValueTag } from "../SqValue";
+
+export const measure = (cb: () => void, times = 1) => {
+  const t1 = new Date();
+
+  for (let i = 1; i <= times; i++) {
+    cb();
+  }
+  const t2 = new Date();
+  return (t2.getTime() - t1.getTime()) / 1000;
+};
+
+export const red = (str: string) => `\x1b[31m${str}\x1b[0m`;
+export const green = (str: string) => `\x1b[32m${str}\x1b[0m`;
+
+export type OutputMode = "NONE" | "RESULT_OR_BINDINGS" | "RESULT_AND_BINDINGS";
+
+export type RunProps = {
+  output: OutputMode;
+  measure?: boolean;
+  sampleCount?: string | number;
+};
+
+export const run = (
+  src: string,
+  props: RunProps = { output: "RESULT_OR_BINDINGS" }
+) => {
+  const project = SqProject.create();
+  if (props.sampleCount && Number(props.sampleCount) !== NaN) {
+    project.setEnvironment({
+      sampleCount: Number(props.sampleCount),
+      xyPointLength: Number(props.sampleCount),
+    });
+  }
+  project.setSource("main", src);
+  const time = measure(() => project.run("main"));
+
+  const bindings = project.getBindings("main");
+  const result = project.getResult("main");
```

Comment:
What happens if the project is run 100 times? Would the results of the last one be shown? This is a bit confusing.